### PR TITLE
HHIOS-349 Add new instructions ui + Lane info

### PR DESCRIPTION
**Pull request by engali94**
### Open Tasks

- [HHIOS-349] 

### Infos for Reviewer
Simulator Screenshot - iPhone 16 - 2024-11-17 at 19 43 02

### CODE REVIEW

**Comment by lafikl** [comment]
can you add a screenshot please for those of us lurking here :)

File: HudHud/Navigation/UI/FerrostarSwiftUI/Views/TopDrawerView.swift [hunk 1]
```diff
@@ -94,17 +77,19 @@ struct TopDrawerView<PersistentContent: View, ExpandedContent: View>: View {
     public var body: some View {
         self.content
             .background(self.backgroundColor)
-            .cornerRadius(12)
-            .shadow(radius: 12)
+            //            .cornerRadius(12)
```
Comment (NaifAlrashed) [inline HudHud/Navigation/UI/FerrostarSwiftUI/Views/TopDrawerView.swift:80]: Better to delete

**Review by NaifAlrashed — APPROVED** [review]
Great work!

**Comment by engali94** [comment]
> can you add a screenshot please for those of us lurking here :)

@lafki updated the description

> can you add a screenshot please for those of us lurking here :)

sure updated the description

**Pull request merged**